assets/js/translator.js

```diff
@@ -447,12 +447,23 @@ function getPairs() /*: JQueryPromise<any> */ {
             }
         });
 
-        for(var srcLang in pairs) {
-            // Default for new users is first available pair
-            curSrcLang = srcLang;
-            curDstLang = pairs[srcLang][0];
-            break;
+        // Default for new users is first available Browser Preference Language pair
+        var prefLang;
+        if(languages[navigator.languages[0]] !== undefined) {
+            prefLang = navigator.languages[0];
+        }
+        else if(languages[navigator.languages[1]] !== undefined) {
+            prefLang = navigator.languages[1];
+        }
+        else if(languages[navigator.languages[2]] !== undefined) {
+            prefLang = navigator.languages[2];
         }
+        else {
+            prefLang = 'en';
```

Comment:
The preference should be the `locale` when available.

Reply:
is it possible that locale will available for all time?

Reply:
done :)..